src/main/kotlin/org/ozwillo/dcimporter/config/ScheduledTask.kt

```diff
@@ -0,0 +1,17 @@
+package org.ozwillo.dcimporter.config
```

Comment:
Not really a config class. Put it in service/job for instance.

Reply:
Ok, moved